linux/vtpm_cold_add_remove/vtpm2_pcr_consistency_test.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Get ESXi host logical processor number"
+  include_tasks: ../../common/esxi_get_cpu_mem_info.yml
+
+- name: "Set fact of the maximum vCPU number of hot add test"
+  ansible.builtin.set_fact:
+    max_num_cpus: "{{ [esxi_logical_cpu_num | int, vm_cpu_hotadd_max | default(16) | int] | min }}"
+
+- name: "Maximum VM vCPU number of CPU hot add test"
+  ansible.builtin.debug: var=max_num_cpus
+
+- name: "Get CPU info"
```

Comment:
Above tasks are not necessary, since you do not do CPU add test, if you just want to change the VM CPU config, just switch the values, for example, CPU number is the same, change the CPU cores per socket number.